code/game/atoms.dm

```diff
@@ -48,6 +48,8 @@ var/global/list/ghdel_profiling = list()
 	//var/harm_label_icon_state //Makes sense to have this, but I can't sprite. May be added later.
 	var/list/last_beamchecks // timings for beam checks.
 
+	var/vending_cat = null// subcategory for vending machines.

```

Comment:
At least move this to movable atoms.
There's no sense having it bloating turf memory.

Though I can't think of a vending machine that gives out anything other than items.